java/api/src/main/java/io/ray/api/call/PyActorCreator.java

```diff
@@ -0,0 +1,24 @@
+package io.ray.api.call;
+
+import io.ray.api.PyActorHandle;
+import io.ray.api.Ray;
+import io.ray.api.function.PyActorClass;
+
+public class PyActorCreator extends ActorCreatorBase<PyActorCreator> {
+  private final PyActorClass pyActorClass;
+  private final Object[] args;
+
+  public PyActorCreator(PyActorClass pyActorClass, Object[] args) {
+    this.pyActorClass = pyActorClass;
+    this.args = args;
+  }
+
+  @Override
+  public PyActorCreator setJvmOptions(String jvmOptions) {
```

Comment:
only put this in `ActorCreator`

Reply:
Python worker doesn't have a `jvm options`, this is used to avoid the user to call this method by mistake